src/commonmark/en/content/implementation/support.md

```diff
@@ -28,58 +27,62 @@ implementation, database development and maintenance. The change log and
 roadmap sections provide links to the relevant pages in the Launchpad
 site described later.
```

Comment:
This is out of date.

We don't really have change logs and roadmap linked from the documentation (probably should add at least a page with links to release notes in the documentation site).